R/db-sql.R

```diff
@@ -150,7 +150,18 @@ sql_table_analyze.DBIConnection <- function(con, table, ...) {
 
 #' @rdname db-sql
 #' @export
-sql_table_index <- function(con, table, columns, name = NULL, unique = FALSE, ...) {
+sql_table_index <- function(con,
+                            table,
+                            columns,
+                            name = NULL,
+                            unique = FALSE,
+                            ...,
+                            call = caller_env()) {
+  check_table_ident(table, call = call)
```

Comment:
A table ident can be anything like `ident()`, `ident_q()`, `in_schema()`, `in_catalog()`, `DBI::Id()` or a string.